haiku-apps/bemobile/bemobile-0.9.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A program to manage contacts and messages stored on your mobile phone"
+DESCRIPTION="BeMobile is a program to manage contacts and messages stored on \
+your mobile phone. It has been developed for Motorola C550 and L6 models but \
+should work with any phone that supports AT command set."
+HOMEPAGE="https://github.com/ytmytm/beos-bemobile"
+COPYRIGHT="2007 Maciej Witkowia"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="fea6464ceb5b570f7a533e59b24a68f9fb093006"
+SOURCE_URI="https://github.com/ytmytm/beos-mobile/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="2c903363e0725556d896ed5a15f68b77a0461b4512efd0106a2c68d8b30d4e5c"
+SOURCE_FILENAME="beos-bemobile-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="beos-bemobile-$srcGitRev"
+ADDITIONAL_FILES="
+	bemobile.pld
+	Makefile
+	Resource.rdef
+	"
+ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	bemobile = $portVersion
+	app:BeMobile = $portVersion
+	"
+REQUIRES="
+        haiku
+        "
+
+BUILD_REQUIRES="
+        haiku_devel
+        "
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+	
+BUILD()
+{
+	cp "$portDir"/additional-files/* .
```

Comment:
Since we also have PR for the makefile at the app's repo, we can remove all patches.